### DISCUSSION TITLE
Apple Pay support

### DISCUSSION THREAD

**Discussion by skibaalex** [ionic-team/capacitor#3170] [Q&A]
Is there anyone that knows about a way to implement Apple Pay and google pay?

**Comment by imjacobclark** [comment]
https://github.com/samkelleher/cordova-plugin-applepay

**Comment by skibaalex** [comment]
@imjacobclark This repo is not supporting the new versions of ios already tried anything else?
would [Apple Pay JS](https://developer.apple.com/documentation/apple_pay_on_the_web/apple_pay_js_api) work?

**Comment by stefanhk31** [comment]
+1 to this issue. Apple Pay support is pretty essential to our app, and the plugin only supports up to iOS 12. Is this something the capacitor team is working on?

**Comment by thomasvidas** [comment]
At the moment there isn't an Apple Pay or Google Pay plugin that the Capacitor team officially supports. You can [create a Capacitor Plugin](https://capacitorjs.com/docs/plugins/creating-plugins) and make a then implement your own native [Apple Pay](https://developer.apple.com/documentation/passkit/apple_pay/) and native [Google Pay](https://developers.google.com/pay/api/android/overview) APIs.

But Ionic is staffing up a plugins team that specifically works on high quality Capacitor plugins so maybe they'll be one in the future. Who knows? 🤔 😄

**Reply by Saqib92** [reply]
I am from Future. still no plugin from capacitor team.

**Reply by robingenz** [reply]
There is Ionic Payments: https://ionic.io/products/payments 
I haven't used it yet but it looks like it is just that.

**Reply by 3adeling** [reply]
@robingenz you have to pay more than $2,499/month to be able to use Ionic Payments!!

**Reply by jcesarmobile** [reply]
In fact the $2,499/month doesn't include Ionic Payments.
If your company qualify for the startup plan it's $249/month and includes Ionic Payments and other enterprise plugins.
https://ionic.io/pricing/indies-and-startups

**Reply by 3adeling** [reply]
@jcesarmobile didn't know about the startup plan, however, I belive ionic payments should be for one time purchase not for a monthly price.

**Reply by jcesarmobile** [reply]
The monthly payment is for the startup plan, which includes enterprise plugins, appflow, etc.
If you only want the payments plugin and not the startup plan you should contact sales, there is a “talk to sales” button on the payments plugin page
https://ionic.io/products/payments

**Reply by marshall86** [reply]
i'm using the enterprise plugin, i'm testing it but i'm struggling to complete the payment flow. I correctly created all the certificates in the apple developer portal and configured the domain but when I call my backend in the authorization url, it doesn't receive any data in POST even though (if i'm not wrong) it should receive the payment data that includes the apple token, signature ecc.. Any idea on what could be wrong?
